lib/stdlib/doc/src/gen_event.xml

```diff
@@ -245,6 +245,16 @@ gen_event:stop             ----->  Module:terminate/2
                   Which term depends on the error.</p></item>
             </list>
           </item>
+          <item>
+              <p>If the calling process exits later, the event manager will
+              delete all handlers the calling process supervises (and call
+              <seemfa marker="#Module:terminate/2">
+              <c>Module:terminate({'EXIT', Pid, Reason}, State)</c></seemfa>),
```

Comment:
Module:terminate/2 callback will not receive {'EXIT', Pid, Reason} but {stop, Reason} instead.

I would suggest skipping argument examples in links - because arguments are described in detail for both Module:terminate/2 and Module:handle_info/2 in the linked sections. Repeated descriptions are harder to maintain.